### PR TITLE
[Backport from 1.6] Give pserve the ability to open server in browser (pserve --browser)

### PR DESCRIPTION
**Pull request by msabramo**
E.g.:

    pserve app.ini --browser

or:

    pserve app.ini -b

### CODE REVIEW

File: pyramid/scripts/pserve.py [hunk 1]
```diff
@@ -334,6 +341,17 @@ def serve():
                     msg = ''
                 self.out('Exiting%s (-v to see traceback)' % msg)
 
+        if self.options.browser:
+            def open_browser():
+                context = loadcontext(SERVER, app_spec, name=app_name, relative_to=base,
+                        global_conf=vars)
+                url = 'http://{host}:{port}/'.format(**context.config())
```
Comment (mmerickel) [inline pyramid/scripts/pserve.py]: shouldn't this always be localhost? the "host" option is specific to an individual wsgi server and is usually the interface to listen on, not the ip address to access. Technically the port is also specific to the wsgi server but probably acceptable.

Comment (msabramo) [inline pyramid/scripts/pserve.py]: Oh I don't know maybe? Host seems to always be 127.0.0.1 or 0.0.0.0 and those both work. 

What problem are you worried about?

Comment (mmerickel) [inline pyramid/scripts/pserve.py]: http://0.0.0.0 works? Are you sure? I'm not so sure.

Comment (msabramo) [inline pyramid/scripts/pserve.py]: I think you're right. 

http://superuser.com/questions/536156/how-do-i-get-0-0-0-0-to-resolve-to-localhost-when-browsing-a-url-that-contains-0

I think it works on my Mac but it seems this is not universal. 

I'll change it.

**Comment by mmerickel** [comment]
Sweet, sorry I didn't see that you had pushed a fix. Github doesn't email me when you push.

**Comment by msabramo** [comment]
No worries. Usually I post a comment when I update a PR but I guess I forgot this time.

**Pull request merged**